### PR TITLE
[FIX] web_drop_target: Hide it when it is not the current action or has no chatter

### PR DESCRIPTION
**Pull request by etobella**
Fixes https://github.com/OCA/web/pull/2066#issuecomment-955672542 on 13.0

### CODE REVIEW

File: web_drop_target/static/src/js/web_drop_target.js [hunk 1]
```diff
@@ -140,7 +140,15 @@ odoo.define("web_drop_target", function(require) {
...
         _onBodyFileDragover: function(ev) {
             ev.preventDefault();
-            if (_.isEmpty(this._get_drop_items(ev))) {
+            const actionManager = this.findAncestor(function(ancestor) {
+                return ancestor instanceof ActionManager;
+            });
+            const controller = actionManager.currentDialogController;
+            if (
+                _.isEmpty(this._get_drop_items(ev)) &&
+                this.renderer.chatter &&
```
Comment (Tardo) [inline web_drop_target/static/src/js/web_drop_target.js]: Won't this make it not work if you don't have a chatter? for example, DMS.

Comment (etobella) [reply web_drop_target/static/src/js/web_drop_target.js]: Ouch!!! Thanks, I think I have a better idea now :wink:

**Comment by victoralmau** [comment]
Can you review it again @Tardo? When this is ready we can unlock other PRs that depend on it. Thank you!

**Comment by victoralmau** [comment]
@etobella can you review it according to similar changes https://github.com/OCA/web/pull/2066 in v14?

**Comment by etobella** [comment]
@victoralmau it is the same commit we find in https://github.com/OCA/web/pull/2066

The other commits are already present on commit history of web_drop_target https://github.com/OCA/web/commits/13.0/web_drop_target

**Review by victoralmau — APPROVED** [review]

**Review by Tardo — APPROVED** [review]

**Pull request merged**